src/main/scala/com/databricks/labs/overwatch/pipeline/Gold.scala

```diff
@@ -359,10 +373,41 @@ class Gold(_workspace: Workspace, _database: Database, _config: Config)
         jobRunCostPotentialFactModule.execute(appendJobRunCostPotentialFactProcess)
         GoldTargets.jobRunCostPotentialFactViewTarget.publish(jobRunCostPotentialFactViewColumnMapping)
       }
+      case OverwatchScope.audit =>
+        config.overwatchScope.foreach {
+          case OverwatchScope.notebooks =>
+            config.overwatchScope.foreach {
+              case OverwatchScope.clusterEvents => {
+                notebookCommandsModule.execute(appendNotebookCommandsProcess)
+                GoldTargets.notebookCommandsTargetView.publish(verboseAuditTargetViewColumnMapping)
+              }
+              case _ =>
+            }
+          case _ =>
+        }
```

Comment:
Please create complex boolean to determine if verboseAuditlogging should be processed and then reference this in a single top level command -- I don't want to nest these case statements like this.

Also make it accessible from both locations where nested cases are used.